packages/stylelint-config/__tests__/__snapshots__/index.js.snap

```diff
@@ -33,11 +33,11 @@ $twoCamelCase: "two";
 
 .selector-1,
 .selector-2,
-.selector-3[type="text"] {
-  background: linear-gradient(#fff, rgb(0 0 0 / 0.8));
+.selector-3[type='text'] {
+  background: linear-gradient(#fff, rgb(0 0 0 / 80%));
   box-sizing: border-box;
-  color: var(--brand-red);
   display: block;
+  color: var(--brand-red);
```

Comment:
This is out of alpha-order, again b/c our rules aren't being respected for some reason